src/sse.ts

```diff
@@ -74,7 +74,7 @@ const onLog = (
   callback: (message: Message) => void,
   senders?: string[]
 ): Unlisten => {
-  const source = `${endpoint('colossus')}/${ctx.account}/${ctx.workspace}/logs?level=${logLevel}`
+  const source = `${colossusEndpoint()}/${ctx.account}/${ctx.workspace}/logs?level=${logLevel}`
```

Comment:
This also needs to change to call the new endpoint for infra services, which in case of colossus would be `infra.io.vtex.com/colossus/v0/...`. This would also need the logic for setting the `upstream-target` header when the cluster configuration is present.